### PR TITLE
Remove VAI provider cloud scheduler support

### LINKED ISSUE #285: Remove Support for Cloud Scheduler
## Overview
Now https://github.com/sky-uk/kfp-operator/issues/284 has been complete the next step is to remove the legacy cloud scheduler code along with the code that was introduced to migrate existing schedules over to native Vertex AI scheduler.

## Notes
- all code should be removed for the cloud scheduler pipeline
- all migration code should be removed
- any configuration/deployment setup for run-intents etc should be removed
- all documentation updated
- any diagram updated with simplified setup

### PR DESCRIPTION
**Pull request by grahamia**
Closes #285 

Following migration from cloud scheduler into native Vertex AI scheduler we can now remove the legacy scheduler code and migration code.

## Tasks:
- [X] update documentation
- [X] update diagrams
- [X] remove cloud scheduler code
- [X] remove migration code

### CODE REVIEW

File: argo/providers/vai/provider.go [hunk 1]
```diff
@@ -36,8 +33,9 @@ var labels = struct {
 	RunConfigurationNamespace string
 	RunName                   string
 	RunNamespace              string
-	LegacyNamespace           string
-	LegacyRunConfiguration    string
+	// TODO SHOULD WE DELETE THESE NOW?
```
Comment (grahamia) [inline argo/providers/vai/provider.go:36]: @jmendesky

Comment (grahamia) [reply argo/providers/vai/provider.go:36]: comment in code is
// For compatability with resources created with v0.3.0 and older

Comment (jmendesky) [reply argo/providers/vai/provider.go:36]: This can be removed now.

File: argo/providers/vai/provider.go [hunk 2]
```diff
@@ -228,7 +217,47 @@ func (vaip VAIProvider) CreateRun(ctx context.Context, providerConfig VAIProvide
...
+	for name, value := range run.RuntimeParameters {
+		parameters[name] = &aiplatformpb.Value{
+			Value: &aiplatformpb.Value_StringValue{
+				StringValue: value,
+			},
+		}
+	}
+
+	pipelineJob := &aiplatformpb.PipelineJob{
+		Labels:         run.Labels,
```
Comment (jmendesky) [inline argo/providers/vai/provider.go:236]: I think the `VAIRun` created above is not necessary. I fact, I think we can remove that struct entirely.

Comment (grahamia) [reply argo/providers/vai/provider.go]: Agreed and have removed, only 2 values used from struct so passing as params now

**Review by jmendesky — APPROVED** [review]
:shipit:

**Pull request merged**